src/inc/clrnt.h

```diff
@@ -835,7 +835,7 @@ RtlVirtualUnwind_Unsafe(
 //  X86
 //
 
-#if defined(_TARGET_X86_)
+#if defined(_TARGET_X86_) && defined(WIN32)
```

Comment:
Should this be "defined(_MSC_VER)" to only enable these pragmas when compiling with VC++ toolset and not just win32 (which maybe true for LLVM based compile, for example)?